doc/user_doc/vic_installation/SUMMARY.md

```diff
@@ -14,12 +14,14 @@
   * [Advanced Examples of Deploying a VCH](vch_installer_examples.md)
   * [Use Different User Accounts for VCH Deployment and Operation](set_up_ops_user.md)
   * [Deploy a VCH with vSphere Integrated Containers Registry (Harbor)](deploy_vch_harbor.md)
-* [Installing the vSphere Web Client Plug-in](install_vic_plugin.md)
-  * [vCenter Server For Windows with a Web Server](plugin_vc_web.md)
-  * [vCenter Server for Windows without a Web Server](plugin_vc_no_web.md)
+* [Installing the vSphere Client Plug-ins](install_vic_plugin.md)
+  * [Install the HTML5 Plug-In on a vCenter Server Appliance](plugin_h5_vsca.md)
+  * [Install the HTML5 Plug-In on vCenter Server for Windows](plugin_h5_vc_web.md)
+  * [vCenter Server for Windows by Using a Web Server](plugin_vc_web.md)
```

Comment:
Yes, this was deliberate, although inconsistent. The topic titles that are included in this file appear in the nav bar on the left of the HTML output. See https://vmware.github.io/vic-product/assets/files/html/latest/vic_installation/index.html.

So, I often truncate the titles in this file, so that more meaningful content appears in the left nav bar, rather than 5 topics that are all look the same. However, I think that you're right that now that we have both clients, the HTML5 vs. Flex distinction is more important than the type of VC. So, I have dropped the "Install the" but included HTML5 or Flex at the beginning of the title, and tried to abbreviate. The full headings appear in https://github.com/stuclem/vic/blob/h5client/doc/user_doc/vic_installation/install_vic_plugin.md.